tests/test_tydiqa_secondary.py

```diff
@@ -15,3 +18,18 @@ def test_prompt():
         "such as wound, ostomy, and continence nursing and burn center care.\n"
     ) in prompt
     assert prompt.endswith("Answer:")
+
+
+def test_promptsource_template():
+    ds_key, sub_key = "tydiqa", "secondary_task"
+    tydiqa_sec_vld_ds = load_dataset(ds_key, sub_key, split="validation", streaming=True)
```

Comment:
`promptsource` also has a helper of dataset loading but I really want to use `streaming=True` if at all possible (depending on each dataset's compression format).